src/jcampEncoder/JcampCreator.js

```diff
@@ -2,21 +2,18 @@
 /**
  * Created by acastillo on 3/2/16.
  */
+const Encoder = require('./VectorEncoder');
+const Integer = {MAX_VALUE: 2e31 - 1, MIN_VALUE: -2e31};
```

Comment:
please use `Number.MAX_SAFE_INTEGER` and `Number.MIN_SAFE_INTEGER`.

BTW 2e31 means 2*10^31. I guess you were thinking about Math.pow(2, 31).